File: TreasurersApp/Models/EmailAddress.cs
namespace TreasurersApp.Models
{
    using Newtonsoft.Json;

    public class EmailAddress
    {
        [JsonProperty("id")]
        public int EmailAddressId { get; set; } // EmailAddressID (Primary key)

        [JsonProperty("email")]
        public string Email { get; set; } // Email (length: 256)

        [JsonProperty("createdBy")]
        public System.Guid CreatedBy { get; set; } // CreatedBy

        [JsonProperty("createdDate")]
        public System.DateTime CreatedDate { get; set; } // CreatedDate

        [JsonProperty("lastModifiedBy")]
        public System.Guid LastModifiedBy { get; set; } // LastModifiedBy

        [JsonProperty("lastModifiedDate")]
        public System.DateTime LastModifiedDate { get; set; } // LastModifiedDate

        // Reverse navigation

        /// <summary>
        /// Child ContributorEmailAddresses where [ContributorEmailAddress].[EmailAddressID] point to this entity (FK_ContributorEmailAddress_EmailAddress)
        /// </summary>
        public System.Collections.Generic.ICollection<ContributorEmailAddress> ContributorEmailAddresses { get; set; } // ContributorEmailAddress.FK_ContributorEmailAddress_EmailAddress

        public EmailAddress()
        {
            ContributorEmailAddresses = new System.Collections.Generic.List<ContributorEmailAddress>();
        }
    }

}
// </auto-generated>

File: TreasurersApp/ClientApp/src/app/models/EmailAddress.ts
export class EmailAddress {
  id: number = null;
  email: string = null;
  createdBy: string = null;
  createdDate: Date = null;
  lastModifiedBy: string = null;
  lastModifiedDate: Date = null;

  constructor(id: number,
              email: string,
              createdBy: string,
              createdDate: Date,
              lastModifiedBy: string,
              lastModifiedDate: Date) {
    this.id = id;
    this.email = email;
    this.createdBy = createdBy;
    this.createdDate = createdDate;
    this.lastModifiedBy = lastModifiedBy;
    this.lastModifiedDate = lastModifiedDate;
  }
}
